br/br-checkpoint-restore.md

```diff
@@ -0,0 +1,63 @@
+
+The log backup metadata contains an array of file metadata. Each file metadata in the array represents a file composed of multiple log backup files. The file metadata records the offset and size of a log backup file in the concatenated file. Therefore, `br` can use the triple `(log backup metadata name, file metadata array offset, log backup file array offset)` to uniquely identify a log backup file.
+
+## Usage limitations
+
+Checkpoint restore relies on the GC mechanism and cannot record all data that has been restored. The following sections provide the details.
+
+### GC will be paused
+
+During a log restore, the order of the restored data is unordered, which means that the deletion record of a key might be restored before its write record. If GC is triggered at this time, all data of the key will be deleted and then GC cannot process subsequent write records of the key. To avoid this situation, `br` pauses GC during log restore. If `br` exits halfway, GC remains paused.
+
+After the log restore is completed, GC is restarted automatically without manual startup. However, if you decide not to continue the restore, you can manually enable GC as follows:
+
+The principle of `br` pausing GC is to execute `SET config tikv gc.ratio-threshold = -1.0` to set `gc.ratio-threshold` to a negative number, thus pausing GC. You can manually enable GC by modifying the value of [`gc.ratio-threshold`](/tikv-configuration-file.md#ratio-threshold). For example, to reset to its default value, you can execute `SET config tikv gc.ratio-threshold = 1.1`.
+
+### Some data needs to be restored again
+
+When `br` retries a restore, some data that has been restored might need to be restored again, including the data being restored and the data not recorded by the checkpoint.
+
+- If the interruption is caused by an error, `br` persists the meta information of the data restored before exit. In this case, only the data being restored needs to be restored again in the next retry.
+
+- If the `br` process is interrupted by the system, `br` cannot persist the meta information of the data restored to the external storage. Since `br` persists the meta information every 30 seconds, data restored in the last 30 seconds before interruption cannot be persisted and needs to be restored again in the next retry.
+
+### Avoid modifying cluster data during the restore
+
+- After a log restore failure, avoid writing, deleting, or creating tables in the cluster. This is because the log backup data might contain DDL operations for renaming tables. If you modify the cluster data, the checkpoint restore cannot confirm whether the deleted or existing table resulted from external operations, which affects the accuracy of the next restore retry.
+- After a snapshot restore failure, it is not recommended to delete tables.
```

Comment:
```suggestion
After a restore failure, avoid writing, deleting, or creating tables in the cluster. This is because the log backup data might contain DDL operations for renaming tables. If you modify the cluster data, the checkpoint restore cannot confirm whether the deleted or existing table resulted from external operations, which affects the accuracy of the next restore retry.
```